install_config/configuring_authentication.adoc

```diff
@@ -21,23 +21,17 @@ access tokens] to authenticate themselves to the API.
 
 As an administrator, you can configure OAuth using the
 link:../install_config/master_node_configuration.html[master configuration file] to specify an
-link:#identity-providers[identity provider]. If you installed OpenShift using
-the
+link:#identity-providers[identity provider].
 ifdef::openshift-enterprise[]
-link:../install_config/install/quick_install.html[Quick Installation] or
-endif::[]
-link:../install_config/install/advanced_install.html[Advanced Installation]
-method, the
-ifdef::openshift-enterprise[]
-link:#DenyAllPasswordIdentityProvider[Deny All]
-endif::[]
-ifdef::openshift-origin[]
-link:#AllowAllPasswordIdentityProvider[Allow All]
-endif::[]
-identity provider is used by default, which denies access for all user names and
-passwords. To allow access, you must choose a different identity provider and
-configure the master configuration file appropriately (located at
+If you installed OpenShift using the
+link:../install_config/install/quick_install.html[Quick Installation]  method,
```

Comment:
again, for the same reason as below. bu if you have any suggestions as to how to reword this and keep the link, i can update it. it's best seen when you look at the compiled docs rather than the diff, and switch between ose and origin